spec/index.html

```diff
@@ -495,25 +505,42 @@ <h3>Blank nodes</h3>
       without using an IRI to <a>identify</a> any particular thing.
       This is not the same as assuming that the blank node indicates an 'unknown' IRI.</p>
 
-    <p>Suppose I is a simple interpretation and A is a mapping from a set of blank nodes
-      to the universe IR of I.
-      Define the mapping [I+A] to be I on <a>names</a>,
-      and A on blank nodes on the set: [I+A](x)=I(x) when x is a <a>name</a>
-      and [I+A](x)=A(x) when x is a blank node;
-      and extend this mapping to triples and RDF graphs using the rules given above for ground graphs.
-      Then the semantic conditions for an RDF graph are:</p>
+    <p>Suppose that</p>
+    <ul>
+      <li>I is a simple interpretation,</li>
+      <li>A is a mapping from a set N of blank nodes to the universe IR of I, and</li>
+      <li class="new">B is a mapping from a subset of N to the set of <a>ground</a> terms, such that for every x in that subset, I(B(x)) = A(x).</li>
+      <li>B is extended to names and quoted triples:
+        <ul>
+          <li>if x is a <a>name</a>, B(x) = x</li>
+          <li>if x is a quoted triple (s, p, o), B(x) = (B(s), B(p), B(o))</li>
```

Comment:
```suggestion
          <li>If x is a <a>name</a>, B(x) = x</li>
          <li>If x is a quoted triple (s, p, o), B(x) = (B(s), B(p), B(o))</li>
```

Reply:
I would rather keep it that way for the moment, as this is the same style as other similar definitions inherited from the RDF 1.1 spec. I generally agree that it improves readability, but for the sake of consistency, we should apply this change globally.

Reply:
I see you actually updated them all. Fine by me then.